Common/Securities/Forex/Forex.cs

```diff
@@ -170,16 +171,44 @@ public static void DecomposeCurrencyPair(string currencyPair, out string baseCur
                 }
             }
 
-            if(bases.Count == 0) 
+            if (bases.Count == 0)
             {
                 throw new ArgumentException($"No base currency found for the pair: {currencyPair}");
             }
             else
-            if (quotes.Count == 0) 
+            if (quotes.Count == 0)
             {
                 throw new ArgumentException($"No quote currency found for the pair: {currencyPair}");
             }
-            
+
+        }
+
+        /// <summary>
+        /// You have currencyPair AB and one known symbol (A or B). This function returns another one (B or A).
+        /// </summary>
+        /// <param name="currencyPair">Currency pair AB</param>
+        /// <param name="knownSymbol">Known part of the currencyPair (either A or B)</param>
+        /// <returns>Returns other part of currencyPair (either B or A)</returns>
+        public static string CurrencyPairDual(string currencyPair, string knownSymbol)
+        {
+            string CurrencyA = null;
+            string CurrencyB = null;
+
+            DecomposeCurrencyPair(currencyPair, out CurrencyA, out CurrencyB);
+
+            if (CurrencyA == knownSymbol)
+            {
+                return CurrencyB;
+            }
+            else
+            if (CurrencyB == knownSymbol)
```

Comment:
code style - please keep `else if` on a single line.

Reply:
Ok, on it fixing both.